marketplace/users/views.py

```diff
@@ -1,3 +1,14 @@
+from django.http import HttpResponseRedirect
+
 from django.shortcuts import render
+from .models import User
+from .form import RegistrationForm
 
 # Create your views here.
+# Create your views here.
+def seller_registration(request):
+    if request.method == 'POST':
+        form = RegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return render(request, 'seller_form.html', {'form': form})
```

Comment:
Here you've only defined the function for `POST` method, what about a `GET` request?